.prettierignore

```diff
@@ -24,3 +24,4 @@ versioned_docs/version-1.0
 versioned_docs/version-1.1
 versioned_docs/version-1.2
 versioned_docs/version-1.3
+optimize_versioned_docs/version-3.7.0
```

Comment:
I added this version of optimize docs to the list of folders that prettier should ignore; unfortunately, I did this _after_ moving a bunch of files, and it looks like there are a bunch of formatting changes included in this PR. 

I'll do my best to inspect all the formatting changes. I'd rather not roll back the moved files & move them again, as it would mean re-updating any broken markdown links....but if there a bunch of bad formatting changes I'll have no choice.

Reply:
Let us know if you need help here.